docs/modules/ROOT/pages/how-tos/configuring/component-nudges.adoc

```diff
@@ -109,3 +109,23 @@ NOTE: You can also use **kubectl edit** to add the build-nudges-ref instead of p
 
 * Ensure that the existing references in the repositories of your components are correct.
 * Try running the necessary commands again, and ensure you have the correct syntax.
+
+== Using custom renovate config
+It is possible to configure your own custom renovate config, which will be used during nudging.
+Renovate config has to be specified in ConfigMap in your namespace.
+
+.There are two possible options:
+. create global config in ConfigMap named `global-nudging-renovate-config` which will be used for all nudged components in your namespace,
+unless component specific config exists
+. create config for specific nudged component in ConfigMap named `nudging-renovate-config-COMPONENT_NAME` and
```

Comment:
Usually `<component-name>` style is used

Reply:
ok